src/System.Collections.Specialized/tests/ListDictionary/GetKeysTests.cs

```diff
@@ -96,7 +93,10 @@ public void Test01()
             {
                 Assert.False(true, string.Format("Error, returned Keys.Count = {0}", ks.Count));
             }
-            arr = Array.CreateInstance(typeof(Object), len);
+
+            Assert.False(ks.IsSynchronized);
```

Comment:
What does this have to do with this test?  I realize these tests are already a mess, but for anything new we add, we should strive to do so in a manner that fits as much as possible with how we'd like to the tests to be.